charts/spire/README.md

```diff
@@ -7,33 +7,76 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 **Homepage:** <https://github.com/spiffe/helm-charts-hardened/tree/main/charts/spire>
 
-## Install notes
+## Install Instructions
 
+### Non Production
 To do a quick non production install suitable for quick testing in something like minikube:
 
 ```shell
-helm install -n spire-server spire-crds spire-crds --repo https://spiffe.github.io/helm-charts-hardened/ --create-namespace
-helm install -n spire-server spire spire --repo https://spiffe.github.io/helm-charts-hardened/
+helm upgrade --install -n spire-server spire-crds spire-crds --repo https://spiffe.github.io/helm-charts-hardened/ --create-namespace
+helm upgrade --install -n spire-server spire spire --repo https://spiffe.github.io/helm-charts-hardened/
 ```
 
-To customize, start with a base values file and edit as needed:
+### Production
+
+Preparing a production deployment requires a few steps.
+
+Step 1: Save the following to your-values.yaml, ideally your git repo.
+```yaml
+global:
+  openshift: false # If running on openshift, set to true
+  spire:
+    recommendations:
+      enabled: true
+    namespaces:
+      create: true
+    ingressControllerType: "" # If not openshift, and want to expose services, set to a supported option [ingress-nginx]
+    # Update these
+    clusterName: example-cluster
+    trustDomain: example.org
+spire-server:
+  ca_subject:
+    # Update these
+    country: ARPA
+    organization: Example
+    common_name: example.org
+```
+
+Step 2: If you need a non default storageClass, append the following to the spire-server section and update:
+```
+  persistence:
+    storageClass: changeme
+```
 
+Step 3: If your Kubernetes cluster is OpenShift based, use the output of the following command for your trustDomain:
 ```shell
-curl -o your-values.yaml https://raw.githubusercontent.com/spiffe/helm-charts-hardened/main/examples/production/example-your-values.yaml
+oc get cm -n openshift-config-managed  console-public -o go-template="{{ .data.consoleURL }}" | sed 's@https://@@; s/^[^.]*\.//'
 ```
 
-Then:
+Step 4: Find any additional values you might want to set based on the documentation below or the examples at:
+https://github.com/spiffe/helm-charts-hardened/tree/main/examples
+
+In particular, consider using an external database.
+
+Step 5: Edit your-values.yaml with the appropriate values.
```

Comment:
This step doesnt seem to have anything under it.

Reply:
The other steps are about gathering the options you might need, and this is talking about actually making the changes.

So, something else above is unclear I guess.